lnwallet/channel.go

```diff
@@ -3441,16 +3409,14 @@ func newHtlcResolution(signer Signer, localChanCfg *channeldb.ChannelConfig,
 	// With the transaction created, we can generate a sign descriptor
 	// that's capable of generating the signature required to spend the
 	// HTLC output using the timeout transaction.
-	htlcCreationScript, err := senderHTLCScript(localKey, remoteKey,
-		revokeKey, htlc.RHash[:])
+	htlcCreationScript, err := senderHTLCScript(keyRing.localKey,
+		keyRing.remoteKey, keyRing.revocationKey, htlc.RHash[:])
 	if err != nil {
 		return nil, err
 	}
-	timeoutTweak := SingleTweakBytes(commitPoint,
-		localChanCfg.PaymentBasePoint)
 	timeoutSignDesc := SignDescriptor{
 		PubKey:        localChanCfg.PaymentBasePoint,
-		SingleTweak:   timeoutTweak,
+		SingleTweak:   keyRing.localKeyTweak,
```

Comment:
The `localDelayTweak` still needs to be added. As is now, this'll be invalid when attempting to sweep outgoing timed out HTLC's.

Reply:
Actually scratch this, this is set properly after all. Just (my own naming) was a bit confusing here :p